SignalServiceKit/protobuf/OWSSignaliOSProtos.proto

```diff
@@ -4,11 +4,11 @@
  * Licensed according to the LICENSE file in this repository.
  */
 
-package signalios;
+// iOS - since we use a modern proto-compiler, we must specify
+// the legacy proto format.
+syntax = "proto2";
 
-// Signal-iOS
-import "objectivec-descriptor.proto";
-option (google.protobuf.objectivec_file_options).class_prefix = "OWSSignaliOSProtos";
+package signalios;
```

Comment:
I also wonder if we should include "Proto" in these package names, e.g. ServiceProto_X, IOSProto_Y, etc. Just an idea.

Reply:
I changed the SignalService.proto package to be camel case so that the generated class names would be like SignalService_Envelope instead of Signalservice_Envelope. 

Beyond ensuring that they compile, I haven't messed with the other .proto files, but I'd imagine a similar change.

Reply:
> I also wonder if we should include "Proto" in these package names, e.g. ServiceProto_X, IOSProto_Y, etc. Just an idea.

Yeah, I like that.

Reply:
Ok - I've included the yet-to-be-used protos and modified the class prefix to include the word Proto, e.g. SignalServiceProtos_Envelope